components/automate-ui/src/app/pages/api-token/list/api-token-list.component.ts

```diff
@@ -1,15 +1,15 @@
-import { Component, EventEmitter, OnInit } from '@angular/core';
```

Comment:
I am guessing you cherry-picked from the tokens PR to get the `pending` function and got this file along with it...? OK to remove this file's changes here.

Reply:
I plan on merging this one after your tokens PR so I can drop this whole commit